package.json

```diff
@@ -69,6 +68,7 @@
   },
   "devDependencies": {
     "@balena/lint": "^8.2.8",
+    "@balena/pinejs": "file:./",
```

Comment:
Nice, I've used `file:<path>` locally for testing but not for anything like this.